src/elements/content-sidebar/versions/__tests__/__snapshots__/VersionsSidebar.test.js.snap

```diff
@@ -37,14 +37,6 @@ exports[`elements/content-sidebar/versions/VersionsSidebar render should show an
         id="be.sidebarVersions.fetchError"
       />
     </InlineError>
-    <div
```

Comment:
This should still be a supported experience for files without versions. Can we investigate why this snapshot was removed?

Reply:
So this is why this was removed: 
This message shows up when there are no versions, which is what it was intended to, I think. It was also displaying when there was an error from the api. I fixed it so that it only appears when the file has no previous versions, which means that this test, which is testing what is displayed with an error, no longer has the message. When a file has no versions, it doesn't result in an error, just an empty versions list.